examples/trading/contracts/demo/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "demo"
+version = "0.0.1"
+
+[dependencies]
+Sui = { local = "../../../../crates/sui-framework/packages/sui-framework" }
```

Comment:
Should we depend on the `git` version of `sui-framework`, so it's easy for people to copy/paste this? (What do we do for other E2E examples?)

Reply:
@amnn it indeed becomes more usable that way, thanks!